catalog/core/catalog-core-metricsplugin/src/main/java/ddf/catalog/metrics/QueryTypeFilterDelegate.java

```diff
@@ -98,6 +102,11 @@ public Boolean propertyIsFuzzy(String propertyName, String literal) {
         return isComparison = true;
     }
 
+    @Override
+    public Boolean propertyIsEqualTo(String functionName, List<Object> arguments, Object literal) {
+        return isFunction = true;
```

Comment:
actually i think its right for once i can split them out `isFunction = true; return isFunction;` i was just following what the other methods were doing.